Tails specific: i2p sould not start per default. At some point we want
some script to start i2p during init if so selected in tails-greeter,
but at the moment users have to start the i2p script manually.

--- chroot.orig/etc/init.d/i2p	2012-09-24 10:05:13.021047615 +0200
+++ chroot/etc/init.d/i2p	2012-09-24 10:03:30.638108333 +0200
@@ -7,2 +7,2 @@
-# Default-Start: 2 3 4 5
+# Default-Start:
-# Default-Stop: 0 1 6
+# Default-Stop: 0 1 6 2 3 4 5
